SmartTutor/ContentModel/LearningObjects/ChallengeModel/FulfillmentStrategy/NameChecker/NamingRule.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace SmartTutor.ContentModel.LearningObjects.ChallengeModel.FulfillmentStrategy.NameChecker
+{
+    public class NamingRule
+    {
+        [Key] public int Id { get; set; }
+        public List<string> BannedWords { get; set; }
+        public List<string> RequiredWords { get; set; }
```

Comment:
To avoid introducing a new table in the database I'll configure EF to serialize and deserialize this as a string separated by commas. Ping me when you're mostly finished so that I can make this final commit for this PR.

Reply:
Understood!